boule/ellipsoid.py

```diff
@@ -146,6 +146,32 @@ def gravity_pole(self):
         )
         return result
 
+    def _prime_vertical_radius(self, sinlat):
```

Comment:
```suggestion
    def prime_vertical_radius(self, sinlat):
```

Might as well make this public if we’re planning to use it in Harmonica.

Reply:
Yes! I was thinking about that after I opened the PR, I completely agree.